### ISSUE TITLE
Not compatible with Spine 3.8.76

### ISSUE THREAD

**Issue by gogoflash** [photonstorm/phaser#4860]
## Version

* Phaser Version: 3.20.1

* Operating system: Win 7 x64

## Description

Incompatible with Spine 3.8.76 animation json files. Throws parse errors on json animation files.

Whats wrong/change in new Spine json files(see in pics below):
1. No "time" properties anymore if it equal 0(default on start) value. Phaser parser "covert" it to NaN.
2. No "x", "y" properties in "scale" blocks if it equals default "1" value. Its has no problem on position where 0 is default. But Phaser parser set it for 0 in common position and scale parser.
3. Different structure for "skins" block.

## Additional Information
I wrote a working solution with backward compatibility with old Spine format(no need to reconvert all of your older animations): [SpineWebGLPlugin.zip](https://github.com/photonstorm/phaser/files/3851537/SpineWebGLPlugin.zip)

## Screens
1) ![2019-11-15_0240](https://user-images.githubusercontent.com/4013058/68906483-7254dc00-0755-11ea-9e09-a4068dfc69cb.png)
2) ![2019-11-15_0243](https://user-images.githubusercontent.com/4013058/68906495-7d0f7100-0755-11ea-8fde-a36738b6bf7b.png)
3) ![2019-11-15_0255](https://user-images.githubusercontent.com/4013058/68907528-2dcb3f80-0759-11ea-85ee-de21c561b46a.png)

**Comment by photonstorm** [comment]
They broke this between 3.8.72 and 3.8.76 ?! Seriously?

**Comment by photonstorm** [comment]
BTW there is no "Phaser parser" - the json is parsed by the Esoteric Spine Runtimes. So, to fix this properly, the runtimes just need to be updated to 3.8.76 and the plugin re-built. Even so, it seems strange they broke the json format this much in a tiny point release. Is this the case @badlogic ? (where can I find a change log between point releases of the runtimes so I can see what else changed between these versions?)

**Comment by badlogic** [comment]
Jupp, that's the case. We don't use semantic versioning. The major version
number is really only increased when there are huge editor side feature
additions (frequency=years). The minor version number is increased on every
new editor release (frequency=month). Exports from differing minor version
numbers are not guaranteed to be compatible, e.g. 3.7.x exports don't work
with the 3.8 runtimes. The patch version is increased everytime we do an
editor side bug fix release.

TL;DR: ignore the major version number, treat minor version numbers as
incompatible to each other, treat patch version numbers as non API/format
breaking (guaranteed).

We have a CHANGELOG.md file in the root of our runtimes repo, and our
release blog posts also discuss breaking changes extensively. I'm out and
about on mobile atm, so please excuse me not providing links.

**Comment by photonstorm** [comment]
@badlogic thanks for clarifying - this makes sense. I think I'll just edit the Phaser docs to state exactly which version of the runtimes the plugin is currently using, and give instructions on how to update them should they want to use newer one. Also, have a look at this: https://github.com/photonstorm/phaser/blob/master/plugins/spine/src/SpinePlugin.js#L377-L397 it's something I have to do otherwise Spine batching breaks. May be worth making part of the official runtime?

**Comment by photonstorm** [comment]
I'm going to close this issue, as I've updated the Spine docs to state exactly what runtime Phaser is compatible with - if you want to use another version, you'll need to replace the runtimes and build the plugin. I'll do this when we release new builds of Phaser, but it will never be fully in sync with the official runtimes.
